pkg/ruler/compat.go

```diff
@@ -70,6 +70,7 @@ func engineQueryFunc(engine *logql.Engine, overrides RulesLimits, checker readyC
 			logproto.FORWARD,
 			0,
 			nil,
+			"",
```

Comment:
Ditto; can we add a `ruler` hint?

Reply:
I didn't understand. what is `ruler` hint?

Reply:
This is code in the `ruler`.